### DISCUSSION TITLE
Using Fn as Win modifier

### DISCUSSION THREAD

**Discussion by jacksongoode** [rxhanson/RectanglePro-Community#191] [Ideas]
It would be great to use the Fn key as the global modifier to tile windows. This implementation is done in [Swish](https://highlyopinionated.co/swish/) as the default modifier and it is quite nice to have a single (often unused key) for this use.

I know this has been mentioned as [difficult here](https://github.com/rxhanson/RectanglePro-Community/issues/18) but I think it would be worthwhile to implement. I'm guessing a listener is being used with Swish:

>  Due to a limitation on Apple's side though, Swish needs to listen to all mouse movement. This averages to well below 1% for most users, so any impact on battery life won't be noticeable.

But it would at least be useful to have this feature as a toggle for folks who want the ability for Fn as a modifier.

**Comment by rxhanson** [comment]
In Rectangle Pro, the Fn key can be used as a modifier with the window throw, quick throw, and move/resize functionality. 

It sounds like you want to record a keyboard shortcut using Fn?

**Reply by rxhanson** [reply]
Ah, missed the subject of this - you specifically want fn+arrow keys for the win arrow keys one, got it. I'll add it to my backlog and see where I can fit this one in.

**Reply by jacksongoode** [reply]
Fantastic, that's great to hear!

**Reply by jacksongoode** [reply]
@rxhanson Just wondering if there's been any update on this? No rush, of course.

**Reply by rxhanson** [reply]
No update yet, I've been working on a new app and I try to focus on one thing at a time. Almost ready to switch back into a Rectangle Pro cycle again, where I'll be reassessing the backlog priority.
